lib/tests/streamlit/commands/query_params_test.py

```diff
@@ -33,15 +33,24 @@ def test_set_query_params_exceptions(self):
         with self.assertRaises(StreamlitAPIException):
             st.experimental_set_query_params(embed_options="show_colored_line")
 
-    def test_get_query_params_after_set_query_params(self):
+    def test_get_query_params_after_set_query_params_single_element(self):
         """Test valid st.set_query_params sends protobuf message."""
+
         p_set = dict(x=["a"])
         st.experimental_set_query_params(**p_set)
         p_get = st.experimental_get_query_params()
+        self.assertEqual(p_get, dict(x="a"))
+
+    def test_get_query_params_after_set_query_params_list(self):
+        """Test valid st.set_query_params sends protobuf message."""
+
+        p_set = dict(x=["a", "b"])
+        st.experimental_set_query_params(**p_set)
+        p_get = st.experimental_get_query_params()
         self.assertEqual(p_get, p_set)
 
     def test_set_query_params_empty_str(self):
         empty_str_params = dict(x=[""])
         st.experimental_set_query_params(**empty_str_params)
         params_get = st.experimental_get_query_params()
-        self.assertEqual(params_get, empty_str_params)
+        self.assertEqual(params_get, dict(x=""))
```

Comment:
nit: not sure why some of these were originally written `dict(x=["a"])` etc, but I don't see why we aren't just writing these as dict-literals.